docs/standard/modelling/identifiers.rst

```diff
@@ -1,142 +1,94 @@
 .. _guidance-identifiers:
 
 Real world identifiers
-=============================
+==================================
 
-Overview
-------------------------
+Real world identifiers are essential for making beneficial ownership data interoperable. By 'real world identifier' we mean reference strings for entities, people and items (issued by authoritative registration schemes) which have gained widespread use in the world. People can use them to verify that the referenced person, entity or item exists.
+In BODS, real world identifiers can be published for:
 
-To create a link between statements, and the real-world organisations and people they relate to, statements may include a range of identifying information. We use a common :any:`Identifier object <schema-identifier>` with the following properties:
+- entities
+- people
+- stock markets and trading exchanges
+- tradable securities
 
-* ``scheme`` should be a value from a codelist of known identifier sources. Separate codelists exist for entities and persons. See below.
+Entity identifiers
+-------------------------------
 
-* ``id`` should be the value assigned to the relevant entity or person in that scheme.
+Use an :ref:`Identifier <schema-identifier>` in the ``identifiers`` array of an Entity Statement to supply a company registration number, Legal Entity Identifier (LEI), or other real world identifier.
 
-* ``uri`` may be used to provide a canonical URI for the entity or person within the scheme.
+The value for ``Identifier.scheme`` SHOULD come from `org-id.guide <http://org-id.guide>`_. This resource contains details of company registers and other identifier sources. If the identifier scheme you need to reference is not already listed on org-id.guide `propose a new entry <https://org-idguide-handbook.readthedocs.io/en/latest/contribute/#proposing-a-new-entry>`_.
 
-* ``schemeName`` should be the name of the list, registry or ID system.
+If it is useful to publish an internal identifier from your data management system, first consider whether it should be used as a :ref:`record identifier <record-identifiers>`. If not, either:
 
-A good-quality Identifier will contain ``scheme`` and ``id`` values which will uniquely identify an entity or person. Where these are not available, ``schemeName`` can be used to refer to the registration system in which the person or entity is known to be represented. When publishing an Identifier object, a value for either ``scheme`` or ``schemeName`` MUST be present.
+- publish your full list of internal identifiers, and propose it as an entry to `org-id.guide <http://org-id.guide>`_, or
```

Comment:
I would either have this bullet list as one sentence or rephrase it so it doesn't have the 'or' at the end of the first bullet.

Reply:
I've left this as is. I feel it's too long to squish into one unformatted sentence. And we need to retain the 'or'.